ios/sdk/src/LaunchOptions.m

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright @ 2018-present Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Intents/Intents.h>
+
+#import <React/RCTBridge.h>
+#import <React/RCTBridgeModule.h>
+
+@interface LaunchOptions : NSObject<RCTBridgeModule>
+
+@property (nonatomic, weak) RCTBridge *bridge;
+
+@end
+
+@implementation LaunchOptions
+
+RCT_EXPORT_MODULE();
+
+- (dispatch_queue_t)methodQueue {
```

Comment:
This is a React Native thing. You can specify on which GCD (Grand Central Dispatch) queue you'd like the code to run. Since we deal with stuff which should (potentially) only be touched from the UI thread, we choose the "main" queue, which is the one running in the UI thread.